wdio.conf.js

```diff
@@ -24,8 +24,6 @@ const config = {
 
   // Configuration for terra-toolkit's SeleniumDocker service
   seleniumDocker: {
-    enabled: !process.env.TRAVIS,
-    cleanup: false,
```

Comment:
Cleanup is not a valid seleniumDocker configuration key. This value is never considered.